src/Rest/RestClient.cs

```diff
@@ -66,16 +66,16 @@ private HttpContent CreateFileStream(RestRequest request)
         private Uri BuildUri(RestRequest request)
         {
             var queryStringBuilder = new StringBuilder();
-            request.QueryParameters.ForEach((p) =>
+            request.QueryParameters.ForEach(p =>
             {
                 queryStringBuilder.Append(queryStringBuilder.Length == 0 ? "?" : "&");
-                queryStringBuilder.Append($"{p.Key}={Uri.EscapeDataString(p.Value.ToString())}");
+                queryStringBuilder.Append($"{p.Key}={Uri.EscapeDataString(p.Value)}");
             });
 
             return new Uri(_baseUrl, request.Resource + queryStringBuilder.ToString());
         }
 
-        public async Task<RestResponse> ExecuteHttpRequestAsync(RestRequest request)
+        internal async Task<RestResponse> ExecuteHttpRequestAsync(RestRequest request)

```

Comment:
technically breaking change but nobody should have used it

Reply:
**the class was already internal** so no one could use it before. actually, I don't even understand this:

```csharp
internal class Myclass
{
  public void MyMethod()
  {
  }
}
```